opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -77,6 +78,7 @@ namespace Opm {
         EclipseState(DeckConstPtr deck);
 
         ScheduleConstPtr getSchedule() const;
+        IOConfigConstPtr getIOConfig() const;
```

Comment:
These `xxxPtr` typedefs are remnants from the time we were using `Boost::shared_ptr<>` - and knew it should go away. The `std::shared_ptr<>` is here to stay - I reccomend just using `std::shared_ptr<IOConfig>` in new code.